 
Data Hora: 29/01/2018 9:42:3
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\PainelControle\PainelControle.ApplicationService\Service\ParametroService.cs:linha 67
   em FrameWork.Excecao.RepositorioChaves.TNucleoPainel.GetTempoExpiracao(Int32 idFIlial) na C:\Users\jnunes\Source\Repos\plus_api_framework\NucleoCompartilhado\RepositorioChaves\TNucleoPainel.cs:linha 52
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 126

--------------------------------------------
 
Data Hora: 29/01/2018 8:44:56
Rotina:    System.Collections.Generic.IList`1[Faturamento.Dominio.Views.DocumentosaidaItemView] SalvarItensInsercao(Faturamento.Dominio.Views.DocumentosaidaView, Faturamento.Dominio.Views.DocumentosaidaView, System.String)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs
Linha:     598
Tipo Erro: 1
Nome Erro: ERRO_NullReferenceException
Título:    Referência de objeto não definida para uma instância de um objeto.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em Faturamento.ApplicationService.Service.DocumentosaidaService.SalvarItensInsercao(DocumentosaidaView entidadePostado, DocumentosaidaView entidade, String view) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 598
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1037
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.Salvar(HttpRequestMessage request, DocumentosaidaView entidade) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 303

--------------------------------------------
 
Data Hora: 29/01/2018 8:45:18
Rotina:    System.Collections.Generic.IList`1[Faturamento.Dominio.Views.DocumentosaidaItemView] SalvarItensInsercao(Faturamento.Dominio.Views.DocumentosaidaView, Faturamento.Dominio.Views.DocumentosaidaView, System.String)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs
Linha:     598
Tipo Erro: 1
Nome Erro: ERRO_NullReferenceException
Título:    Referência de objeto não definida para uma instância de um objeto.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em Faturamento.ApplicationService.Service.DocumentosaidaService.SalvarItensInsercao(DocumentosaidaView entidadePostado, DocumentosaidaView entidade, String view) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 598
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1037
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.Salvar(HttpRequestMessage request, DocumentosaidaView entidade) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 303

--------------------------------------------
